.github/workflows/build-publish.yaml

```diff
@@ -0,0 +1,36 @@
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone source code
+        uses: actions/checkout@v1
+        with:
+          fetch-depth: 1
+      - name: Get version and SHA
+        id: version-sha
+        run: |
+          echo "##[set-output name=version;]$(cat VERSION)"
```

Comment:
curious: why do we set it in steps variables as we only need these constants in the docker build section ?

Reply:
This was leftover from when I was using `cd` and the folder approach, but I suppose they can be combined now.